<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="GBOX_seq_file">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Sequencer File</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="LBL_seq_file">
           <property name="text">
            <string>Sequencer file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="TXT_filename">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BTN_file_load">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius: 5px ;background-color: rgb(120, 120, 120);  color: rgb(255, 255, 255); color: rgb(255, 255, 255); border-style: outset; border-bottom: 1px solid; border-right: 1px solid</string>
           </property>
           <property name="text">
            <string>Load</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BTN_file_run">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius: 5px ;background-color: rgb(120, 120, 120);  color: rgb(255, 255, 255); color: rgb(255, 255, 255); border-style: outset; border-bottom: 1px solid; border-right: 1px solid</string>
           </property>
           <property name="text">
            <string>Run</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Parameters</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTableWidget" name="TBL_parameters">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="textElideMode">
            <enum>Qt::ElideMiddle</enum>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>100</number>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>150</number>
           </attribute>
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <row>
            <property name="text">
             <string>NUM_REPEAT</string>
            </property>
            <property name="textAlignment">
             <set>AlignLeading|AlignVCenter</set>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Current Value</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>New Value</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string/>
            </property>
            <property name="textAlignment">
             <set>AlignLeading|AlignVCenter</set>
            </property>
           </item>
           <item row="0" column="1">
            <property name="text">
             <string/>
            </property>
            <property name="textAlignment">
             <set>AlignLeading|AlignVCenter</set>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>External Loops</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,3">
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>Device</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QComboBox" name="CBOX_device"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>Parameter</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QComboBox" name="CBOX_parameter"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_8">
           <property name="title">
            <string>Sweep</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QCheckBox" name="CHBOX_sweep">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="CBOX_sweep_option_0"/>
            </item>
            <item>
             <widget class="QComboBox" name="CBOX_sweep_option_1"/>
            </item>
            <item>
             <widget class="QLineEdit" name="TXT_sweep">
              <property name="text">
               <string>ini:fin:num</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="GBOX_fpga">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>105</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>FPGA Settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="5" rowspan="2">
          <widget class="QPushButton" name="BTN_fpga_connect">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius: 5px ;background-color: rgb(120, 120, 120);  color: rgb(255, 255, 255); color: rgb(255, 255, 255); border-style: outset; border-bottom: 1px solid; border-right: 1px solid</string>
           </property>
           <property name="text">
            <string>    Connect    </string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="LBL_fpga_serial">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Serial</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="3" colspan="2">
          <widget class="QLineEdit" name="TXT_serial">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit { border: none;
background-color:rgb(0,0,0);
color:rgb(255, 255, 255); }</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="LBL_fpga_com">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>COM</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="3" colspan="2">
          <widget class="QLineEdit" name="TXT_com">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit { border: none;
background-color:rgb(0,0,0);
color:rgb(255, 255, 255); }</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="4">
          <widget class="QPushButton" name="BTN_hardware_definition">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius: 5px ;background-color: rgb(120, 120, 120);  color: rgb(255, 255, 255); color: rgb(255, 255, 255); border-style: outset; border-bottom: 1px solid; border-right: 1px solid</string>
           </property>
           <property name="text">
            <string>Open Hardware Definition</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="GBOX_progress">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Progress</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="GBOX_log">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Log</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QPlainTextEdit" name="TXT_log">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="undoRedoEnabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string/>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::NoTextInteraction</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Notification</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="NON_mail">
           <property name="text">
            <string>E-mail:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="TXT_mail"/>
         </item>
         <item>
          <widget class="QCheckBox" name="CHBOX_mail">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="GBOX_data">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Data File Settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="2" column="5">
          <widget class="QPushButton" name="BTN_data_path">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial Black</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius: 5px ;background-color: rgb(120, 120, 120);  color: rgb(255, 255, 255); color: rgb(255, 255, 255); border-style: outset; border-bottom: 1px solid; border-right: 1px solid</string>
           </property>
           <property name="text">
            <string>Load path</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="LBL_com_2">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>File name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="3" colspan="3">
          <widget class="QLineEdit" name="TXT_data_filename">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit { border: none;
background-color:rgb(0,0,0);
color:rgb(255, 255, 255); }</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="LBL_data_path">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>File path</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="3" colspan="2">
          <widget class="QLineEdit" name="TXT_data_path">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit { border: none;
background-color:rgb(0,0,0);
color:rgb(255, 255, 255); }</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="4">
          <widget class="QCheckBox" name="CHBOX_individual">
           <property name="text">
            <string>Save iteration files individually</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="4">
          <widget class="QCheckBox" name="CHBOX_save">
           <property name="text">
            <string>Save File</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BTN_file_load</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonFileLoadPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>1063</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_file_run</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonFileRunPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>983</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_fpga_connect</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonFPGAConnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>761</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>859</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_data_path</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonDataLoadPathPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>839</x>
     <y>633</y>
    </hint>
    <hint type="destinationlabel">
     <x>911</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CBOX_device</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>selectComboDevice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>649</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>642</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CHBOX_sweep</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>checkDeviceSweep()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>645</y>
    </hint>
    <hint type="destinationlabel">
     <x>1059</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CHBOX_individual</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>checkIndividualSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>839</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>906</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BTN_hardware_definition</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pressedHardwareDefinition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>648</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>1694</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonFileRunPressed()</slot>
  <slot>buttonFileLoadPressed()</slot>
  <slot>buttonFPGAConnPressed()</slot>
  <slot>buttonDataLoadPathPressed()</slot>
  <slot>selectComboDevice()</slot>
  <slot>checkDeviceSweep()</slot>
  <slot>checkIndividualSave()</slot>
  <slot>pressedHardwareDefinition()</slot>
 </slots>
</ui>
